Assets/Scripts/DeliveryExperiment.cs

```diff
@@ -605,7 +614,7 @@ private IEnumerator ExperimentCoroutine()
 
         #if !UNITY_WEBGL // WebGL DLL
             // LC: ELEMEM
-            if (HOSPITAL_COURIER)
+            if (EFR_COURIER)
```

Comment:
I don't think this if statement needs to be here. The exit message can get sent for any experiment that uses elemem.